cloud-info/cloud-info-wrapper.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/sh
 
+EXTRA_OPTS=""
+
+if test "x${DEBUG}" = "x1"; then
```

Comment:
Honestly I always find this test quite ugly.
Usually I prefer things relying on `-z` and `-n`.
So if you want to rely test the DEBUG value something like
```shell
if [ -n "$DEBUG" ] && [ "$DEBUG" = "1" ]; then
  EXTRA_OPTS="$EXTRA_OPTS --debug"
fi
```

Reply:
and if we don't care about the exact value of `$DEBUG` but just want it to be set, `[ -n "$DEBUG" ]` may be enough.

Reply:
I guess we do not really care about the value, so test -n should be ok, I'm committing a change